buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/license/RemoteLicenseFetcher.groovy

```diff
@@ -44,14 +50,15 @@ abstract class RemoteLicenseFetcher implements Serializable {
    */
   public final String getText() {
     IOException storedEx = null
+    List<String> urls = remoteUrls
 
     for (int i = 0; i < 3; ++i) {
       try {
         if (i > 0) {
           Thread.sleep(i * 1000)
         }
 
-        return getTextAttempt()
+        return getTextAttempt(urls[i % urls.size()])
```

Comment:
hm, looks like there is no guarantee that the license will be fetched from the url that is specified in the pom, is there? If not, this may be a problem

@ashwinraghav wdyt?

Reply:
True. If this is a real concern, a middle ground could be make fetchers deal with either `http` or `https` as equals, which will reduce somewhat the amount of fetchers without actual merging different sources.